.travis.yml

```diff
@@ -1,30 +1,23 @@
 sudo: false
 language: python
-python:
-    - "2.7"
-    - "3.4"
```

Comment:
The language versions are intended to be used for a testing matrix, rather than "these are the list of versions I want installed in the same image", but I agree sometimes the Travis testing matrix isn't very flexible. Treeherder has to go to some great contortions to get the exact environment we need for each test suite (we abuse the matrix include option quite horrifically [link](https://github.com/mozilla/treeherder/blob/master/.travis.yml)).